Files/Interacts/Interaction.cs

```diff
@@ -439,6 +439,7 @@ public async Task<bool> OpenPath(string path, FilesystemItemType? itemType = nul
         // TODO: Split this function to call OpenFile() and OpenDirectory() separately
         {
             bool isHiddenItem = NativeFileOperationsHelper.HasFileAttribute(path, System.IO.FileAttributes.Hidden);
```

Comment:
Note that hidden items don't work with StorageFile so `fileExists` would always be false for hidden items

Reply:
How do I check for that?

Reply:
For folders there's `CheckFolderAccessWithWin32` should be almost the same for files

Reply:
Ok, I got a small workaround this

Reply:
In case you need it ;)
- to distinguish hidden files from hidden folders
`isFolder = NativeFileOperationsHelper.HasFileAttribute(filePath, System.IO.FileAttributes.Directory)`
- to distinguish a shortcut to a file from a shortcut to a folder
`isShortcutToFolder = (bool)response.Message["IsFolder"]`